libs/SalesforceSDKCore/SalesforceSDKCoreTests/SalesforceOAuthUnitTestsCoordinatorDelegate.m

```diff
@@ -51,4 +51,12 @@ - (void)oauthCoordinator:(SFOAuthCoordinator *)coordinator didFailWithError:(NSE
     XCTFail(@"user agent authentication flow should not fail");
 }
 
+- (void)oauthCoordinator:(SFOAuthCoordinator *)coordinator didBeginAuthenticationWithSafariViewController:(SFSafariViewController *)svc {
+    [self log:SFLogLevelDebug format:@"%@ called.", NSStringFromSelector(_cmd)];
```

Comment:
These look like they should implement some form of the `XCTFail` calls implemented above.  These methods should never be called.